ParisBNF/abb/BNFabb88.xml

```diff
@@ -0,0 +1,210 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb88" xml:lang="en" type="mss">
```

Comment:
Nafisa, you took the wrong number. Abbadie 88 exists, but its description is on on p 194-95 of Conti Rossini's catalogue, numbered 120. The proper signature is alwas under the description. Your manuscript should have been Abbadie 106 (no. 88 of the catalogue, with my mark). Since the xml:id of the manuscript should be the same as the file name, you have to created a new file BNFabb106 with xml:id="BNFabb106", to accomodate the description which is now here. Instead, here you do the true Abbadie 88. 
There are some other things to be corrected, I will come tomorrow and explain you.